homeassistant/components/twinkly/light.py

```diff
@@ -7,6 +7,7 @@
 from typing import Any
 
 from aiohttp import ClientError
+from packaging import version
```

Comment:
Why is this not using awesomeversion like all other integrations/parts?

Reply:
Just because I did not know all other integrations use that.  
I'll add another PR to switch

Reply:
https://github.com/home-assistant/core/pull/83205

Reply:
Great 👍